### PR TITLE
New LosslessAegisInsights

### PR DESCRIPTION
**Pull request by khanti42**
Closes: #754

### CODE REVIEW

File: src/registry.json [hunk 1]
```diff
@@ -1542,6 +1542,30 @@
...
+      "metadata": {
+        "name": "Lossless Aegis Insights",
+        "author": {
+          "name": "Lossless",
+          "website": "https://aegis.lossless.io/"
+        },
+        "website": "https://aegis.lossless.io/snap/",
+        "summary": "Lossless Aegis Insights is a companion to Lossless Aegis, which enhances your Web3 safety by thoroughly scanning smart contracts before you interact with them. This ensures a secure and risk-free experience for your transactions, swaps, claims, and mints.",
+        "description": "Lossless Aegis Insights is an add-on designed to analyze smart contracts without the need to interact with them. It provides reports and risk scores that help you assess the risks associated with both known and unknown smart contracts, helping you keep your funds safe.\nFeatures:\n- Easy setup\n- Wallet security, through transaction analysis\n- Get a risk score, assessment before committing a transaction\n- Get initial analysis of the address you will be interacting with.",
+        "category": "security",
```
Comment (FrederikBolding) [inline src/registry.json:1556]:
```suggestion
        "category": "transaction insights",
```
CI doesn't support the new category name here yet

File: src/registry.json [hunk 2]
```diff
@@ -1542,6 +1542,30 @@
...
+        "support": {
+          "contact": "mailto:hello@lossless.io",
+          "faq": "https://aegis.lossless.io/snap/"
+        },
+        "sourceCode": "https://github.com/Lossless-Cash/aegis-snap/"
+      },
+      "versions": {
+        "0.1.2": {
```
Comment (FrederikBolding) [inline src/registry.json:1564]: This version does not exist on NPM

Comment (khanti42) [reply src/registry.json:1564]: Checking with the dev. He put this version in the request but pushed only version 0.2.0.

Comment (khanti42) [reply src/registry.json]: Confirmed by the dev, it should be 0.2.0.

**Review by Montoya — APPROVED** [review]

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**